config.py

```diff
@@ -713,6 +713,24 @@
         citation_needed_templates = [
             '출처'
         ],
+    ),
+    
+    ur = dict(
+        lang_name = 'اردو',
+        lang_dir = 'rtl',
+        database = 'urwiki_p',
+        
+        wikipedia_domain = 'ur.wikipedia.org',
+        citation_needed_category = 'ماخذ_میں_نامکمل_اندراجات',
+        beginners_link = 'https://ur.m.wikipedia.org/wiki/ویکیپیڈیا:حوالہ_دہی',
+        beginners_link_title = 'ویکیپیڈیا:حوالہ دہی',
+        lead_section_policy_link = '',
+        lead_section_policy_link_title = '',
+        citation_needed_templates = [
+            'حوالہ_درکار',
```

Comment:
Please use a space rather than an underscore: https://travis-ci.org/eggpi/citationhunt/builds/463840146#L554

Reply:
Glad to hear you. Please modify this patch as you wish, Thanks. :)